### PR TITLE
Task #3 - Fudi website

### PR DESCRIPTION
**Pull request by pavlobaranchuk**
Please see the screenshot attached to this comment. 

The mobile version is available via the following link: https://user-images.githubusercontent.com/17711411/46116724-24ee7600-c206-11e8-8be0-6d770cf91ec0.jpg

fudi_website

### CODE REVIEW

**Review by IYeskov — APPROVED** [review]

**Comment by ihor4568** [comment]
Please add all your teammates as reviewers. You need only 2 approves - but everyone should be able to see

File: .jsbeautifyrc [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
```
Comment (ihor4568) [inline .jsbeautifyrc:1]: why 4 spaces indent?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+          </a>
+          <a class="browse__link " href="/ "></a>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="statistic ">
+    <div class="container statistic__content ">
+      <div class="statistic__item ">
```
Comment (ihor4568) [inline index.html]: why extra space at the end of `class` Could you please fix such cases?

File: index.html [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+        <h3 class="statistic__number ">3,182</h3>
+        <p class="statistic__name text ">Spices and Herbs </p>
+      </div>
+    </div>
+  </section>
+
+  <hr class="footer__line ">
+  <footer class="footer ">
+    <div class="container footer__content ">
```
Comment (ihor4568) [inline index.html]: same - extra space at the end. Please review your code and fix such cases.

File: .jsbeautifyrc (hunk 1 above)
Comment (pavlobaranchuk) [reply .jsbeautifyrc:1]: Fixed.
P.S. I am always adding all team-members as reviewers it's just hard to push them to actually accept the invitation if you are not in the office (will be on Monday).

File: index.html (hunk 2 above)
Comment (pavlobaranchuk) [reply index.html]: Fixed. Thank you for noticing.

**Pull request merged**